apps/vendor/models.py

```diff
@@ -0,0 +1,161 @@
+    is_sub_vendor = models.BooleanField(null=True, default=None)
+    firm_status_id = models.ForeignKey(FirmStatuses, on_delete=models.CASCADE, null=True, default=None, db_column='firm_status_id')
+    territory_id = models.ForeignKey(Territory,  on_delete=models.CASCADE,null=True, default=None, db_column='territory_id')
+    vendor_category_id = models.ForeignKey(VendorCategory, on_delete=models.CASCADE, null=True, default=None, db_column='vendor_category_id')
+    contact_person = models.CharField(max_length=255, null=True, default=None)
+    picture = models.ImageField(upload_to=custom_upload_to, max_length=255, null=True, default=None)
+    gst = models.CharField(max_length=255, null=True, default=None)
+    registration_date = models.DateField(null=True, default=None)
+    cin = models.CharField(max_length=255, null=True, default=None)
+    pan = models.CharField(max_length=255, null=True, default=None)
+    gst_category_id = models.ForeignKey(GstCategories, on_delete=models.CASCADE, null=True, default=None, db_column='gst_category_id')
+    gst_suspend = models.BooleanField(null=True, default=None)
+    TAX_TYPE_CHOICES = [('inclusive', 'Inclusive'),
+                        ('exclusive', 'Exclusive')
+                        ]
+    tax_type = models.CharField(max_length=20, choices=TAX_TYPE_CHOICES , default= 'Inclusive')
+    distance = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    tds_on_gst_applicable = models.BooleanField(null=True, default=None)
+    tds_applicable = models.BooleanField(null=True, default=None)
+    website = models.URLField(max_length=255, null=True, default=None)
+    facebook = models.URLField(max_length=255, null=True, default=None)
+    skype = models.URLField(max_length=255, null=True, default=None)
+    twitter = models.URLField(max_length=255, null=True, default=None)
+    linked_in = models.URLField(max_length=255, null=True, default=None)
+    payment_term_id = models.ForeignKey(VendorPaymentTerms,on_delete=models.CASCADE, null=True, default=None, db_column='payment_term_id')
+    price_category_id = models.ForeignKey(PriceCategories, on_delete=models.CASCADE, null=True, default=None, db_column='price_category_id')
+    vendor_agent_id = models.ForeignKey(VendorAgent, on_delete=models.CASCADE, null=True, default=None, db_column='vendor_agent_id')
+    transporter_id = models.ForeignKey(Transporters, on_delete=models.CASCADE, null=True, default=None, db_column='transporter_id')
+    credit_limit = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    max_credit_days = models.IntegerField(null=True, default=None)
+    interest_rate_yearly = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    rtgs_ifsc_code = models.CharField(max_length=255, null=True, default=None)
+    accounts_number = models.CharField(max_length=255, null=True, default=None)
```

Comment:
Can you check with pavan and apply the encrypt/decrypt that he applied to account number field, Also change variable name to account_number

Reply:
updated Thulasi